styleguide/foundations/colours.stories.mdx

```diff
@@ -69,16 +69,18 @@ Use the ‘Input border’ for interactive elements like input boxes so it meets
 
 <Description>
   {colourLanguageTable([
-    ['Button', 'primary-button-colour'],
-    ['Button hover', 'primary-button-hover-colour'],
+    ['Button', 'button__colour--primary'],
+    ['Button hover', 'button__colour--primary-hover'],
   ])}
 </Description>
 
 ### Adviser
 
 Only use ‘Adviser’ colours for adviser-related components, like ‘Adviser callout’. This is also the brand colour for AdviserNet.
 
-<Description>{colourLanguageTable([['Adviser', 'brand-adviser']])}</Description>
+<Description>
+  {colourLanguageTable([['Adviser', 'brand--adviser']])}
+</Description>
 
```

Comment:
I'm tempted to add `$cads-language__error__colour` and `$cads-language__success__colour` in the documentation but I'm not sure if this needs to be agreed with designers first.